narwhal/source_code_sharing/update_liferay_dxp_source_code_of_hotfixes.sh

```diff
@@ -301,11 +329,15 @@ function print_help {
 }
 
 function process_argument_ignore_zip_files {
+	IGNORE_ZIP_FILES_PRESISTENT_FILE="$(dirname "$(readlink /proc/$$/fd/255 2>/dev/null)")/ignore_zip_files_persistent.txt"
```

Comment:
This is the file in the git repository, right? Are you sure that its path can be fixed this way? I am not sure, because I think this is valid only if one starts the script from that very directory (it is not the case on jenkins).